selfdrive/car/toyota/interface.py

```diff
@@ -270,7 +270,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     if ret.enableGasInterceptor:
       set_long_tune(ret.longitudinalTuning, LongTunes.PEDAL)
     elif candidate in [CAR.COROLLA_TSS2, CAR.COROLLAH_TSS2, CAR.RAV4_TSS2, CAR.RAV4H_TSS2, CAR.LEXUS_NX_TSS2,
-                       CAR.HIGHLANDER_TSS2, CAR.HIGHLANDERH_TSS2, CAR.PRIUS_TSS2]:
+                       CAR.HIGHLANDER_TSS2, CAR.HIGHLANDERH_TSS2, CAR.PRIUS_TSS2, CAR.AVALON_TSS2]:
```

Comment:
Is adding all new TSS2 cars to this tune the plan or should it be a separate PR? I made no verifications or gathered any data.

Reply:
think this should be the default now, right @HaraldSchafer?

Reply:
It isn't yet, I have not merged that PR yet.

Reply:
merged